sudokupad-bet-tree.user.js

```diff
@@ -17,130 +17,15 @@
 window.addEventListener('DOMContentLoaded', () => {
 	const treeMarker = 't';
 
-	const renderTree = (renderer, scaleCoeff, offsetX, offsetY) => {
```

Comment:
The job of this function was to ensure that the bet tree itself and the button's icon look exactly the same, and I'd like to leave it this way.